index.Rmd

```diff
@@ -117,14 +116,37 @@ race_map <- tract_race %>%
   st_transform("+proj=longlat +datum=WGS84") %>%
   leaflet() %>%
   setView(lng = -78.878738, lat = 42.880230, zoom = 7) %>%
-  setMaxBounds(lng1 = -78.878738, lat1 = 45.027500, lng2 = -78.878738, lat2 = 40.440624) %>%
   addProviderTiles("Esri") %>% 
   addPolygons(weight = 2, fillOpacity = 0.8, color = ~ race_pal(estimate), 
               label = ~paste0("Estimate Population: ", estimate)) %>%
   addLegend(position = "topright", pal = race_pal , values = ~estimate,
             title = "Black Population", opacity = 0.7)
 race_map
 ```
+```{r newmapoption}
+race_pal <- colorBin(palette = "YlGnBu", bins = 5, domain = tract_race$estimate)
+income_pal <- colorBin(palette = "Greens", bins = 6, domain = tract_income$estimate)
+tract_race_tnsf=st_transform(tract_race,"+proj=longlat +datum=WGS84")
+tract_income_tnsf=st_transform(tract_income,"+proj=longlat +datum=WGS84")
+newmap=leaflet() %>%
+  addTiles() %>%
+  addPolygons(data=tract_income_tnsf,
+              weight=2,
+              fillOpacity =0.8,
+              color = ~income_pal(estimate),
+              label = ~paste0("Estimate Household Income: ", dollar(estimate)),
+              group="Tract Income") %>%
+  addPolygons(data=tract_race_tnsf,weight = 2, fillOpacity = 0.8, color = ~ race_pal(estimate), 
+              label = ~paste0("Estimate Population: ", ~estimate,
+              group="Tract Race")) %>%
+  addLegend(data = tract_income_tnsf, position = "topright", pal = income_pal , values = ~estimate,
+            title = "Median Household Income", opacity = 0.7,
+            labFormat = labelFormat(prefix = "$")) %>%
+  addLegend(data=tract_race_tnsf,position = "topright", pal = race_pal , values = ~estimate,
+            title = "Black Population", opacity = 0.7) %>%
+  addLayersControl(overlayGroups = c("Tract Income","Tract Race"),position='bottomleft')
```

Comment:
You may need to play with this a bit if you want to use it. It should allow you to switch between the two map layers if you don't want to bother with a side by side

Reply:
Yeah, I will be playing around with the layers a bit to see what looks best, especially once I get my DEC data. Thank you for this feedback!